metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -1,15 +1,23 @@
 import logging
 import traceback
-from collections import defaultdict
 from dataclasses import dataclass
-from textwrap import dedent
-from typing import TYPE_CHECKING, Any, Dict, Iterable, List, Optional, Set, Tuple, Union
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    DefaultDict,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Set,
+    Tuple,
+    Union,
+)
 
 import pydantic
 from pydantic.class_validators import validator
-from sqlalchemy import create_engine, sql
-from sqlalchemy.engine.reflection import Inspector
-from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.engine.reflection import Inspector  # type: ignore
+from sqlalchemy.exc import ProgrammingError  # type: ignore
```

Comment:
Why do we have `type: ignore` all over the place?

Unless there's a very good reason, we shouldn't have type ignores.